Components/Navigation.html

```diff
@@ -0,0 +1,171 @@
+                </a>
+                <a href="/Components/Card.html">
+                    <li class="sidebar__item">Card</li>
+                </a>
+                <a href="/Components/Input.html">
+                    <li class="sidebar__item">Input</li>
+                </a>
+                <a href="/Components/Modal.html">
+                    <li class="sidebar__item">Modal</li>
+                </a>
+                <a href="/Components/Navigation.html">
+                    <li class="sidebar__item">Navigation</li>
+                </a>
+                <a href="/Components/Toast.html">
+                    <li class="sidebar__item">Toast</li>
+                </a>
+            </ul>
+        </div>
+
+        <!-- Nav-bar-component -->
+        <!-- Nav-bar-component Intro -->
+        <div class="Nav-bar-component card">
+            <div class="card-demo Nav-bar-component__intro">
+                <h1 class="card-demo--title Nav-bar-component__intro--title">Navigation</h1>
+                <p class="card-demo--desc Nav-bar-component__intro--desc"><b>Navigation Bar</b> is used to navigate users from one page to different pages of Website and App.</p>
+            </div>
+            
+            <!-- Nav-bar-component on Avatar -->
+            <div class="card-demo Nav-bar-component__content">
+                <h1 class="card-demo--title Nav-bar-component__header--container">Navigation-bar</h1>
+
+                <div class="card-demo--container Nav-bar-component__img--container">
+                    <div class="nav-bar-component__wrapper">
+                        <div class="nav-bar-component--primary-actions">
+                            <h1 class="nav-bar-component__logo">Logo</h1>
+                        </div>
+                        <div class="nav-bar-component--seconday-actions">
+                            <ul class="nav-bar-component__list">
+                                <li class="nav-bar-component__item">Home</li>
+                                <li class="nav-bar-component__item">About Us</li>
+                                <li class="nav-bar-component__item">Contact Us</li>
+                                <li class="nav-bar-component__item">Projects</li>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+                <div class="card-demo--container badge__code--container">
+                    <div class="code-snippet-container ">
+                        <textarea class="code-snippet" name="code-snippet" rows="4" cols="" readonly>&lt;div class=&quot;nav-bar-component__wrapper&quot;&gt;
+    &lt;div class=&quot;nav-bar-component--primary-actions&quot;&gt;
+        &lt;h1 class=&quot;nav-bar-component__logo&quot;&gt;Logo&lt;/h1&gt;
+    &lt;/div&gt;
+    &lt;div class=&quot;nav-bar-component--seconday-actions&quot;&gt;
+        &lt;ul class=&quot;nav-bar-component__list&quot;&gt;
```

Comment:
for code, you can use carbon or thiscodeworks

Reply:
Yes, I could use but I found that both are not responsive.